featuregate/gates.go

```diff
@@ -19,14 +19,54 @@ import (
 	"sync"
 )
 
+// Stage represents the Gate's lifecycle and what is the expected state of it.
+type Stage int8
+
+const (
+	// Alpha is used when creating a new feature and the Gate must be explicitly enabled
+	// by the operator.
+	//
+	// The Gate will be disabled by default.
+	Alpha Stage = iota
+	// Beta is used when the feature flag is well tested and is enabled by default,
+	// but can be disabled by a Gate.
+	//
+	// The Gate will be enabled by default.
+	Beta
+	// Stable is used when feature is permanently enabled and can not be disabled by a Gate.
+	// This value is used to provide feedback to the operator that the gate will be removed in the next version.
+	//
+	// The Gate will be enabled by default and will return an error if modified.
+	Stable
```

Comment:
I am not sure I understand how the operator bits work. Could you elaborate? How does this work?

Reply:
I believe I made reference to wrong persona but the featuregate package guidelines on gate state.

Once a flag is marked stable, it can not be modified but is still until the next release so if someone is referencing the flag. The code should error on parsing the flags.

That is what I understood when reading the readme.